workerd/config.capnp

```diff
@@ -13,6 +13,6 @@ const worker :Workerd.Worker = (
   bindings = [
     ( name = "baseline", json = embed "../data/baseline.json" )
   ],
-  compatibilityDate = "2024-05-03",
-  compatibilityFlags = ["experimental", "nodejs_compat_v2"]
+  compatibilityDate = "2024-09-23",
```

Comment:
can you make this compat date the same everywhere please? the one below is newer.

Reply:
The reason is that wrangler.toml would accept "today" while this has to be an actual compatibility date.
Will add auto-update a follow-up.
Fixing manually for now.